plasma_framework/contracts/src/exits/payment/PaymentExitDataModel.sol

```diff
@@ -18,7 +18,7 @@ library PaymentExitDataModel {
      */
     struct StandardExit {
         bool exitable;
```

Comment:
consider move this `excitable to be front of `exitTarget` so it can be packed together into one 32 bytes slot.